Tasks/Xcode/task.json

```diff
@@ -226,6 +231,55 @@
             "visibleRule": "signStyle = auto"
         },
         {
+            "name": "platform",
```

Comment:
"testPlatform" to make it more YAML friendly?

Reply:
`platformOption` I think would be more like the rest.

Reply:
I could see making it obvious that this setting is highly relevant to testing. However the Xcode man page doesn't say that -destination is just for testing, though I haven't found another use yet. Also we may be able to use `platform` for more in the future (e.g. to set the sdk argument).

Reply:
I see both sides. I'm fine with switching to `destinationPlatformOption` or `testPlatformOption`  ("Option" per Luke's comment about our yaml conventions).